packages/react-magma-dom/src/components/TreeView/TreeView.stories.tsx

```diff
@@ -805,4 +805,40 @@ FlatTree.args = {
   selectable: TreeViewSelectable.off,
 };
 
-FlatTree.parameters = { controls: { exclude: ['isInverse'] } };
\ No newline at end of file
+FlatTree.parameters = { controls: { exclude: ['isInverse'] } };
+
+export const ParentsAndChildrenNotAutoChecked = args => {
+  return <>
+    <TreeView {...args} selectable={TreeViewSelectable.multi}>
```

Comment:
We can remove `selectable` here, and add it to the default props below so it can be an interactive example

Reply:
But new props only work for multi selectable treeview tho

Reply:
but ok, will add

Reply:
Done